core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -169,6 +169,12 @@ private[spark] class DAGScheduler(
 
   private[scheduler] val activeJobs = new HashSet[ActiveJob]
 
+  // Job groups that are canceled with `cancelFutureJobs` as true, with at most
+  // `CANCELLED_JOB_GROUP_SET_SIZE` stored. On a new job submission, if the job group is in this
+  // set, the job will be immediately canceled.
```

Comment:
Spark isn't perfectly consistent about "cancelled" vs "canceled" (two 'L's or one) (and it [seems this is a British vs. American english difference](https://www.merriam-webster.com/grammar/canceled-or-cancelled)), but it looks like Spark's existing user-facing messages generally use the two-L version, e.g. at

https://github.com/apache/spark/blob/d908a212ef6144338f075354b4636a13f1172478/core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala#L2730

Given this, I think we should aim for consistency with the existing user-facing messages and use "cancelled" throughout.

Reply:
I will update the PR to all use double l.